tests/e2e/util.go

```diff
@@ -1168,6 +1168,18 @@ func invokeVCenterServiceControl(command, service, host string) error {
 	return nil
 }
 
+//checks if vCenter has a particular FSS enabled or
```

Comment:
recommended format of the comment,
// <func_name> <purpose>

FSS enabled or 'not'